ext/storage_sources/s3_store/s3_store.cxx

```diff
@@ -69,28 +220,70 @@ s3_customize_file_system(WT_STORAGE_SOURCE *storage_source, WT_SESSION *session,
   const char *bucket_name, const char *auth_token, const char *config,
   WT_FILE_SYSTEM **file_systemp)
 {
+    S3_STORAGE *s3;
     S3_FILE_SYSTEM *fs;
     int ret;
+    WT_CONFIG_ITEM cachedir;
+    WT_FILE_SYSTEM *wt_fs;
+    const char *p;
+    char buf[1024];
+    s3 = (S3_STORAGE *)storage_source;
 
     /* Mark parameters as unused for now, until implemented. */
-    UNUSED(session);
-    UNUSED(bucket_name);
     UNUSED(auth_token);
-    UNUSED(config);
 
     Aws::S3Crt::ClientConfiguration aws_config;
```

Comment:
Use WT_ERR and other macros for error handling.

Reply:
Note: I am pretty sure WT macros wont be available in an extension, since this code is external to main WT sources.